examples/Localization/localization_example_lorenz96.jl

```diff
@@ -0,0 +1,130 @@
+using Distributions
+using LinearAlgebra
+using Random
+
+using Plots
+
+using EnsembleKalmanProcesses
+using EnsembleKalmanProcesses.ParameterDistributions
+using EnsembleKalmanProcesses.Localizers
+import EnsembleKalmanProcesses: construct_mean, construct_cov, construct_sigma_ensemble
+const EKP = EnsembleKalmanProcesses
+
+function rk4(f::Function, y0::Array{Float64, 1}, t0::Float64, t1::Float64, h::Float64; inplace::Bool = true)
```

Comment:
```suggestion
function rk4(f::F, y0::Array{Float64, 1}, t0::Float64, t1::Float64, h::Float64; inplace::Bool = true) where {F}
```
This will force specialization on `rk4` for the particular method `f`.